packages/dds/matrix/package.json

```diff
@@ -103,10 +103,16 @@
 	"c8": {
 		"all": true,
 		"cache-dir": "nyc/.cache",
+		"exclude": [
+			"src/test/**/*.*ts",
+			"dist/test/**/*.*js",
+			"lib/test/**/*.*js"
+		],
```

Comment:
Huh, so this was potentially messing up coverage calculations for this package? Do we have results from before and after? Just curious.

Reply:
Since when running test for an individual package we run: "test:mocha:esm", that is why lib files needs to be included for coverage to be reported correctly.
Also added in the description.